features/desktop/unhandled_errors.feature

```diff
@@ -39,7 +39,7 @@ Feature: Reporting unhandled events
     And the event "device.runtimeVersions.dotnetApiCompatibility" is not null
     And custom metadata is included in the event
     And the stack frame methods should match:
-      | Main+<DoAsyncTest>d__52.MoveNext() | Main+<DoAsyncTest>d__50.MoveNext() | Main.DoAsyncTest() |
+      | Main+<DoAsyncTest>d__54.MoveNext() | Main+<DoAsyncTest>d__52.MoveNext() | Main.DoAsyncTest() |
```

Comment:
Is there any way to make this part a wild card? These numbers can change at any time it seems 🤷

Reply:
I had the same thought.  I can see it's very annoying that an unrelated test starts failing pretty much whenever you do anything.

Reply:
I've raised PLAT-8987 to look into this separately (but soon), so that we can get the pipeline passing again asap.